### PR TITLE
Add js examples for interact with existing browsers

### PR DESCRIPTION
**Pull request by wjwjtf**

### CODE REVIEW

File: .stylelintrc.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module.exports = {
-  extends: 'stylelint-config-ship-shape',
```
Comment (colriot) [inline .stylelintrc.js:2]: It seems to me that you IDE setup has inconsistent linting/formatting setup — can you please rollback all the changes to the files that are unrelated to your PR.

In general it's a good practice to look at the diff or your changes before committing and pushing them to make sure only intended changes will end up in this PR

File: javascript-sdk/examples/interact-with-external-or-existing-browser/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+# Example script: login into websites
+
+This is an example shows how to login into websites by retrieving and interacting with web elements in AgentQL.
+
+## Run the script
+
+- [Install AgentQL SDK](https://docs.agentql.com/javascript-sdk/installation)
+- Save this javascript file locally as **log-into-sites.js**
```
Comment (colriot) [inline javascript-sdk/examples/interact-with-external-or-existing-browser/README.md:8]:
```suggestion
- Save this JavaScript file locally as **log-into-sites.js**
```

File: javascript-sdk/examples/interact-with-external-or-existing-browser/README.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+```bash
+node log-into-sites.js
+```
+
+## Play with the query
+
+Install the [AgentQL Debugger Chrome extension](https://docs.agentql.com/installation/chrome-extension-installation) to play with the AgentQL query. [Learn more about the AgentQL query language](https://docs.agentql.com/agentql-query/query-intro)
+
+# Example script: interact with external or existing browser
```
Comment (colriot) [inline javascript-sdk/examples/interact-with-external-or-existing-browser/README.md:19]: You have 2 blocks with 2 different scripts described. Pls, fix it.

The second bock has no links in markdown, seems like you copied plain text only

File: javascript-sdk/examples/interact-with-external-or-existing-browser/interact-with-external-or-existing-browser.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+const STOCK_QUERY = `
+{
+  number_in_stock
+}
+`;
+
+async function interactWithNewPageInLocalBrowser() {
+  //"""This function demonstrates how to open and interact with a new page your local browser."""
+  const browser = await chromium.connectOverCDP(WEBSOCKET_URL);
+  const page = wrap(await browser.contexts()[0].newPage());
```
Comment (colriot) [inline javascript-sdk/examples/interact-with-external-or-existing-browser/interact-with-external-or-existing-browser.js:23]:
```suggestion
  const page = wrap(await browser.newPage());
```

**Review by colriot — CHANGES_REQUESTED** [review]

**Comment by colriot** [comment]
Can you update the name of the PR to have a valid messaging

File: javascript-sdk/examples/interact-with-external-or-existing-browser/README.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+This is an example shows how to interact with external or existing browser by retrieving and interacting with web elements in AgentQL.
+
+## Run the script
+
+- Install AgentQL SDK
+- Save this JS file locally as interact-with-external-or-existing-browser.js
```
Comment (colriot) [inline javascript-sdk/examples/interact-with-external-or-existing-browser/README.md:26]: Pls, check out the formatting of the original README in Python folder and make sure it is similar. You lost formatting and links
```suggestion
- Save this JS file locally as **interact-with-external-or-existing-browser.js**
```

File: javascript-sdk/examples/interact-with-external-or-existing-browser/interact-with-external-or-existing-browser.js [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+`;
+
+const STOCK_QUERY = `
+{
+  number_in_stock
+}
+`;
+
+async function interactWithNewPageInLocalBrowser() {
+  //"""This function demonstrates how to open and interact with a new page your local browser."""
```
Comment (colriot) [inline javascript-sdk/examples/interact-with-external-or-existing-browser/interact-with-external-or-existing-browser.js:21]: Triple quotes — Python style comments, JS uses it's own style. Also. make sure you preserve comments from original python file along the code itself

```suggestion
// This function demonstrates how to open and interact with a new page your local browser.
async function interactWithNewPageInLocalBrowser() {
```

**Review by colriot — CHANGES_REQUESTED** [review]
Some more things to fix

**Comment by frankfeng98** [comment]
@wjwjtf Hi Wenjie! Let me know if u need help with this :)

**Comment by frankfeng98** [comment]
Closing this PR as it contains unrelated file changes. I will update the example

**Pull request closed without merging**